packages/core/src/extensions/tiptap-extensions/Link/link.test.ts

```
@@ -0,0 +1,855 @@
+    const doc = view.state.doc;
+
+    let textStart = 0;
+    let textEnd = 0;
+    doc.descendants((node, pos) => {
+      if (node.isText && node.text === "click here") {
+        textStart = pos;
+        textEnd = pos + node.nodeSize;
+      }
+    });
+
+    const { TextSelection } = require("@tiptap/pm/state");
```

ESLint

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] Avoid calling `expect` conditionally`

(jest/no-conditional-expect)